docs/UserGuide.md

```diff
@@ -411,38 +411,67 @@ you can simply enter `undone 2` into the command line based on the index labelle
 * The index **must be found in your assignment list**
 </div>
 
-### Scheduling assignments:
+### Scheduling assignments: `schedule`
 
-Schedules the assignment identified by the `INDEX` number used in the displayed assignment list.
-The suggested schedule with start time and end time will be displayed in the assignment card.
-It is guaranteed that the suggested time slot will not class with any of your lessons or the suggested time
+Format: `schedule INDEX expected/EXPECTED_HOUR dobefore/BEFORE_DATE BEFORE_TIME`
+
+You can schedule your assignments from the assignment list by specifying the assignment `INDEX` as shown in your assignment list, 
```

Comment:
Looks good! But I think it'll be good to mention some of the user benefits / problem that schedule solves

Reply:
I agree with @printinghelloworld . Perhaps mentioning how the user will not be required to manually plan a time to do the assignment would be good :)